javascript/javascript1/week3/seriesDurationOfMyLife.js

```diff
@@ -0,0 +1,58 @@
+const seriesDurations = [
+    {
+        title: "Game of thrones",
+        days: 3,
+        hours: 1,
+        minutes: 0,
+    },
+    {
+        title: "Silo",
+        days: 0,
+        hours: 16,
+        minutes: 40,
+    },
+    {
+        title: "Friends",
+        days: 3,
+        hours: 11,
+        minutes: 36,
+    },
+    {
+        title: "How I Met Your Mother",
+        days: 3,
+        hours: 4,
+        minutes: 16,
+    }
+];
+
+function howMuchTimeHaveIWasted(listOfShows) {
```

Comment:
These series percentages are higher than they should be! Can you tell why and fix the bug? 🐛

Reply:
yeah now that i checked it is a lot higher than it should be. like a lot
I am counting each show more than once. thats why it is too high.